cernopendata/modules/collections/descriptions.py

```diff
@@ -105,6 +117,14 @@
         'name': 'CMS Simulated Datasets',
         'text': 'This collection contains CMS Simulated Datasets.'
     },
+    'cms-tools': {
+        'name': 'CMS Tools',
+        'text': ''  # TOFIX Add description
+    },
+    'cms-trigger-information': {
+        'name': 'CMS Trigger Information',
+        'text': ''  # TOFIX Add description
```

Comment:
The collection descriptions could be taken from the COD2 `qa` branch, see file
`invenio_opendata/base/fixtures/websearch.py`.  If we need them at all, actually... since we want to hide the notion of "collections" in COD3?!

Reply:
Actually I don't really see the point of having descriptions as we always redirect to the search  page. WDYT ?